src/pages/AppLogin.vue

```diff
@@ -104,7 +97,7 @@ onMounted(async () => {
             >
               <v-card-button
                 class="sso-button"
-                @click.stop="launchLogin(SocialLoginType.google)"
+                @click.stop="launchLogin('google')"
```

Comment:
@makylfang Can we re-export `SocialLoginType` from the auth SDK as well? It's useful to have this type when calling `loginWithSocial`.

Reply:
Noted.